scripts/testing/runtest

```diff
@@ -152,6 +152,9 @@ def _options():
     options.args    = config.get('default', 'args').split()
     options.targets = resolve_targets(args[1:])
 
+    if options.report is None:
+        options.report = config.get('default', 'report')
```

Comment:
`config.get` will fail if `report` does not exist in the section. Use `config.get('default', 'report', fallback = None)`

You can even change the the whole block by setting `fallback` to `options.report`

Reply:
Uh... `make check` calls the script directly. That caught me by surprise (but it does indeed fail, as demonstrated y the external CI).

> You can even change the the whole block by setting `fallback` to `options.report`

I don't think I can do that: I want to use the config file value if and only if it exists *and* there is no report option passed on the CLI. I'll fix it up with the `fallback = None`.

Reply:
So this is now implemented as I intended it, but I'm open to discussing whether that intention is the right one or not. Do we agree that command-line options should take precedence over config file options?

Reply:
CLI option should take precedence over configuration file.